modules/kinesis/src/it/resources/localstack/01-createStreams.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+export AWS_ACCESS_KEY_ID=foo
+export AWS_SECRET_ACCESS_KEY=bar
+
+aws --endpoint-url=http://localhost:4566 kinesis create-stream --stream-name raw-count --shard-count 1 --region eu-central-1
+aws --endpoint-url=http://localhost:4566 kinesis create-stream --stream-name enriched-count --shard-count 1 --region eu-central-1
+aws --endpoint-url=http://localhost:4566 kinesis create-stream --stream-name bad-1-count --shard-count 1 --region eu-central-1
+
+aws --endpoint-url=http://localhost:4566 kinesis create-stream --stream-name raw-stop --shard-count 1 --region eu-central-1
+aws --endpoint-url=http://localhost:4566 kinesis create-stream --stream-name enriched-stop --shard-count 1 --region eu-central-1
+aws --endpoint-url=http://localhost:4566 kinesis create-stream --stream-name bad-1-stop --shard-count 1 --region eu-central-1
```

Comment:
With the existing tests it would be fine to have only 1 stream of each. If we use a different app name in `count` and `stop` they should consume the streams independently (different KCL DynamoDB tables). I'll update it.

> after each case

The tests are to be run in parallel to save time.